### PR TITLE
[IMP] tumaker_custom: new objects in partner. Industry Sector and Applications.

### PR DESCRIPTION
**Pull request by agaldona**

### CODE REVIEW

**Comment by agaldona** [comment]
@avanzosc/developers  Review when possible please. Thanks!

File: tumaker_custom/security/.~lock.ir.model.access.csv# [hunk 1]
```diff
@@ -0,0 +1 @@
+,avanzosc,avanzosc-HP-Compaq-8100-Elite-SFF-PC,17.02.2017 09:16,file:///home/avanzosc/.config/libreoffice/4;
```
Comment (esthermm) [inline tumaker_custom/security/.~lock.ir.model.access.csv#]: ¿y esto?

**Review by esthermm — APPROVED** [review]

**Review by Daniel-CA — APPROVED** [review]
:+1:

**Review by oihane — APPROVED** [review]
LGTM

**Pull request merged**